app/models/tax.rb

```diff
@@ -0,0 +1,3 @@
+class Tax < ApplicationRecord
+  belongs_to :car
```

Comment:
We agreed that Taxes and Cars should have `has_many through` relationship. Read here https://guides.rubyonrails.org/association_basics.html#the-has-many-through-association

Reply:
I can help you with that Georgi if you want more help with that issue pm me in slack 😊